app/templates/frameworks/legal_authority_no.html

```diff
@@ -0,0 +1,33 @@
+{% extends "_base_page.html" %}
+
+{% block pageTitle %}
+    {{ framework.name }} You cannot sign the Framework Agreement – Digital Marketplace
+{% endblock %}
+
+{% block breadcrumb %}
+    {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Digital Marketplace",
+        "href": "/"
+      },
+      {
+        "text": "Your account",
+        "href": url_for('.dashboard')
+      },
+      {
+        "text": framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
+      },
+      {
+        "text": "Sign agreement"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block mainContent %}
+    <h1 class="govuk-heading-xl">You cannot sign the Framework Agreement</h1>
+    <p class="govuk-body">Only someone with legal authority in your company, like a director or company secretary, can sign the Framework Agreement.</p>
```

Comment:
I wonder if this is an appropriate use case for the [warning text component](https://design-system.service.gov.uk/components/warning-text/).

Reply:
Talked myself out of it - we're not really warning the user about an action they can take.